tests/sdk/test_graph.py

```diff
@@ -14,6 +16,29 @@ def node_graph_linear():
     return {key: [key - 1] if key > 0 else list() for key in range(10)}
 
 
+@pytest.fixture
+def node_graph_straight_branch():
```

Comment:
where is it used ?

Reply:
Let's keep this for now as we don't if it will be used or not.